.gitignore

```diff
@@ -26,6 +26,7 @@
 .env.production.local
 .vscode
 .idea
+.nvmrc
```

Comment:
Actually, go ahead and commit it.
I'm a happy user of nvm as well :slightly_smiling_face:.

Reply:
fyi: version is 10.15.3, and all tests pass with that